eras/conway/impl/src/Cardano/Ledger/Conway/Translation.hs

```diff
@@ -160,7 +161,7 @@ instance Crypto c => TranslateEra (ConwayEra c) API.UTxO where
 
 instance Crypto c => TranslateEra (ConwayEra c) API.ProposedPPUpdates where
   translateEra _ctxt (API.ProposedPPUpdates ppup) =
-    pure $ API.ProposedPPUpdates $ fmap (upgradePParamsUpdate def) ppup -- TODO: Pick UpgradeConwayPParams from ConwayGenesis instead
+    pure $ API.ProposedPPUpdates $ fmap (upgradePParamsUpdate def) ppup
```

Comment:
Is this supposed to use `def`?

Reply:
Yes, I think it does even in `ShelleyGenesis`, but I'm not sure. I was hoping to get a review comment about it. :slightly_smiling_face: